lametro/templates/event/_event_header_live_public_comment.html

```diff
@@ -1,5 +1,5 @@
 <section class="row">
-    {% if has_agenda and not event.has_passed %}
+    {% if has_agenda and event_notices %}
```

Comment:
This branch is allowing for a notice to be posted even when an event has_passed. Because of this, the styling conditionals have been adjusted to allow space for notices if they exist at all.

Reply:
Also something to note, any notice showing up is still contingent on the event having an agenda, to mimic the way public comment instructions only displayed if they had one as well. Lmk if we want to change this!